src/pkg/lint/validate.go

```diff
@@ -142,38 +122,37 @@ func (pkg ZarfPackage) Validate() error {
 			groupedComponents[component.DeprecatedGroup] = append(groupedComponents[component.DeprecatedGroup], component.Name)
 		}
 	}
-
 	for groupKey, componentNames := range groupedComponents {
 		if len(componentNames) == 1 {
 			err = errors.Join(err, fmt.Errorf(lang.PkgValidateErrGroupOneComponent, groupKey, componentNames[0]))
 		}
 	}
-
 	return err
 }
 
-func (a ZarfComponentActions) validate() error {
+// ValidateActions validates the actions of a component.
+func ValidateActions(a v1alpha1.ZarfComponentActions) error {
```

Comment:
Do we need to update any other function than ValidatePackage?

Reply:
Nope